buildout.cfg

```diff
@@ -13,6 +13,24 @@ versions = versions
 [versions]
 # keep in sync with requirements.txt!
 setuptools = 39.1.0
-zc.buildout = 2.12.2
+zc.buildout = 2.13.2
```

Comment:
setuptools and zc.buildout should go away from the version pins IMHO, I think the plonetest buildout already has them.

Reply:
Firstly buildout needs to be sync with requirements.txt, secondly wouldn't being specific for the versions be a good thing so that we don't need to find out suddenly that the future versions of buildout is incompatible for the package?  Or that someone has a older version of buildout and having the versions listed allow him to avoid in traversing through dependency hell?

Reply:
Yes, that is why I was proposing to also make requirements conditional (see comment above)